Expensify/App
# [HOLD] Add "engines" field, .nvmrc and .npmrc to enforce a specific node version

cc @cead22 @stitesExpensify 

### Details
This would help us enforce the node version for both internal and external contributors. 

### Fixed Issues

$ GH_LINK

### Tests/QA Tests
1. Open a terminal in the project's dir
2. Run `nvm install 12`. This will set your node version to 12.
3. Run `npm install`. It should fail, telling you that you're not using the node version specified in the `engines.node` field of the `package.json` file.
4. Run `nvm use`. This will set your node version to the one defined in the `.nvmrc` file. It should be the same version as the one in the `engines.node` field.
5. Run `npm install`. It should install all the packages.
